### DISCUSSION TITLE
macOS default backups location

### DISCUSSION THREAD

**Discussion by lhagan** [standardnotes/forum#2522] [General]
I noticed that Standard Notes on macOS by default backs up to "Standard Notes Backups" in the user's home folder. This is not consistent with macOS conventions -- in general, applications shouldn't be adding files to the home folder unless specifically asked to do so by the user. See [Don’t Pollute User Space](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPFileSystem/Articles/WhereToPutFiles.html#//apple_ref/doc/uid/TP40001411-111946) in Apple's developer documentation. A better location is probably `~/Library/Application Support` -- this is where most per-application backups seem to be on my machine at least.

I know it's easy enough as a user to fix this by changing the backup directory location, but ideally this would not be necessary.

**Comment by atmoio** [comment]
Thanks, I'll take a look at this. We're using home dir for now as it's a safe cross-platform place for it. Ideally we'd have a custom location depending on OS.

**Comment by ghost** [comment]
Five years down the road and this has yet to be resolved. See #3166 which I recently submitted. Rio,
